main/menu.ts

```diff
@@ -20,14 +20,28 @@ export default function setMenu(win: Electron.BrowserWindow) {
                     label: 'Undo',
                     accelerator: 'CmdOrCtrl+Z',
                     click: () => {
-                        win.webContents.send('nyaovim:exec-commands', ['undo']);
+                        const focusedWebContents = webContents.getFocusedWebContents();
+                        if (win.webContents === focusedWebContents) {
```

Comment:
Oh, I didn't consider `<webview>`. You're correct.

Well, I think `win.webContents.isFocused()` is available rather than comparing `WebContents` instance.

https://github.com/electron/electron/blob/master/docs/api/web-contents.md#contentsisfocused